src/main/java/graphql/schema/idl/SchemaGenerator.java

```diff
@@ -183,8 +185,12 @@ public boolean isUseAppliedDirectivesOnly() {
             return useAppliedDirectivesOnly;
         }
 
+        public boolean getAddOnErrorDirective() {
+            return addOnErrorDirective;
+        }
+
         public static Options defaultOptions() {
-            return new Options(true, true, false);
+            return new Options(true, true, false, false);
```

Comment:
Should this default to true - if we are going to add this directive I think it should be on by default (so it can be experimentally used) and if some one wants to opt out, then can

Reply:
SchemaGenerator is the way MOST schemas are built btw